.github/workflows/ci.yml

```
@@ -44,6 +44,8 @@ jobs:
           POSTHOG_SUPPORT: true
           POSTHOG_API_KEY: ${{ secrets.MAINNET_POSTHOG_API_KEY }}
           POSTHOG_HOSTNAME_HTTP: ${{ secrets.MAINNET_POSTHOG_HOSTNAME_HTTP }}
+          GATSBY_GTM_SUPPORT: true
+          GATSBY_GTM_ID: ${{ secrets.GTM_ID }}
```

Non-blocking:

If you agree with the above then

```suggestion
          GOOGLE_TAG_MANAGER_SUPPORT: true
          GOOGLE_TAG_MANAGER_ID: ${{ secrets.GOOGLE_TAG_MANAGER_ID }}
```

---

Same as above